tests/test_outcome.py

```diff
@@ -0,0 +1,25 @@
+from roulette import Outcome
+
+from unittest import TestCase
+
+
+class TestOutcome(TestCase):
+    def setUp(self):
+        self.oc1 = Outcome("Red", 1)
+        self.oc2 = Outcome("Red", 1)
+        self.oc3 = Outcome("Black", 2)
+
+    def test_compare_objects_with_identical_names(self):
+        self.assertEquals(self.oc1, self.oc2)
+
+    def test_same_hash_values_for_equal_objects(self):
+        oc1_hash_value = self.oc1.__hash__()
+        oc2_hash_value = self.oc2.__hash__()
```

Comment:
to improve readability, better use `hash()` function instead of directly invoking magic method `__hash__`.